### DISCUSSION TITLE
Reference to the group as a entry (ex. using UUID) ??

### DISCUSSION THREAD

**Discussion by Nubyua** [keepassxreboot/keepassxc#11509] [Q&A] [answered]
Hello, to all
Is it possible to create an entry as a reference to a group using the group's UUID? 
I have created two groups of passwords: account (e.g. John Doe) and hardware (Dell e7270). 
The "account" group contains entries assigned to a person, and the "hardware" group contains entries assigned to the computer (e.g. licenses, key, passwords). 
When a person leaves company, I close his account, change accesses changing entries in account group, and the computer goes to the new employee (change entries in the hardware group). I would like the hardware group to have a link to the person who uses it and vice versa, and the account group to have a link to the hardware group to know which computer it has access to. There may be several
I know, I can create a hardware subgroup in the account group, but to change entries, e.g. passwords in each piece of equipment, I would have to go through people (and remember who has what and has access to what). It's better for me to create a hardware group and an account.
Any idea?

**Comment by droidmonkey** [comment]
I have read your post three times and I can't understand what you are actually wanting to do. KeePassXC is not a relational database or an asset management system.

**Comment by michaelk83** [comment, marked as answer]
If I'm reading this right, you want a link from one entry to another entry, not to a group of entries. For example, from Employee1 to Workstation1 and vice versa.

AFAIK, there are no links to the entry itself, but you can reference fields within another entry. For example, you can reference the employee's username, or the Workstation's identifier. This will copy the value of that field from the other entry to the current one.
https://keepassxc.org/docs/KeePassXC_UserGuide#_entry_cross_reference

You can place that data in any field you like, and you can add custom fields for that if needed.
https://keepassxc.org/docs/KeePassXC_UserGuide#_additional_attributes
You can also just copy the entry name manually to avoid the complexities of field references.

Finally, there are now tags, which you can add to multiple entries from different groups.
https://keepassxc.org/docs/KeePassXC_UserGuide#_application_layout
https://keepassxc.org/docs/KeePassXC_UserGuide#_adding_an_entry
